contracts/test/VBep20Harness.sol

```diff
@@ -7,7 +7,7 @@ import "../VBep20Delegate.sol";
 import "../Governance/AccessControlManager.sol";
 import "./ComptrollerScenario.sol";
 
-contract VBep20Harness is VBep20Immutable {
+contract VBEP20Harness is VBep20Immutable {
```

Comment:
Why though?

Reply:
It resolves a typescript compile error originating from generated typechain that complains filename casing doesn't match imports